examples/deadline/Local-Zone/python/package/lib/service_tier.py

```diff
@@ -0,0 +1,151 @@
+        :param stack_id: The ID of this construct.
+        :param props: The properties for this construct.
+        :param kwargs: Any kwargs that need to be passed on to the parent class.
+        """
+        super().__init__(scope, stack_id, **kwargs)
+
+        self.version = VersionQuery(
+            self,
+            'Version',
+            version=props.deadline_version
+        )
+
+        # We are excluding the local zones from the Repository. This construct will create an
+        # EFS filesystem and DocDB cluster, both of which aren't available in any local zones at this time.
+        repository_subnets = SubnetSelection(
+            availability_zones=props.availability_zones,
+            subnet_type=SubnetType.PRIVATE
+        )
+        repository = Repository(
+            self,
+            'Repository',
+            vpc=props.vpc,
+            repository_installation_timeout=Duration.minutes(20),
+            version=self.version,
+            vpc_subnets=repository_subnets
+        )
+
+        images = ThinkboxDockerImages(
+            self,
+            'Images',
+            version=self.version,
+            user_aws_thinkbox_eula_acceptance=props.accept_aws_thinkbox_eula
+        )
+
+        server_cert = X509CertificatePem(
+            self,
+            'RQCert',
+            subject=DistinguishedName(
+                cn=f'renderqueue.{props.dns_zone.zone_name}',
+                o='RFDK-Sample',
+                ou='RenderQueueExternal'
+            ),
+            signing_certificate=props.root_ca
+        )
+
+        # The render queue is also put only in the standard availability zones. The service itself
+        # is run in a single zone, while the load balancer that sits in front of it can be provided
+        # all the standard zones we're using.
+        render_queue_subnets = SubnetSelection(
+            availability_zones=[props.availability_zones[0]],
+            subnet_type=SubnetType.PRIVATE
+        )
+        render_queue_alb_subnets = SubnetSelection(
+            availability_zones=props.availability_zones,
+            subnet_type=SubnetType.PRIVATE,
+            one_per_az=True,
+        )
+        self.render_queue = RenderQueue(
+            self,
+            'RenderQueue',
+            vpc=props.vpc,
+            images=images,
+            repository=repository,
+            hostname=RenderQueueHostNameProps(
+                hostname='renderqueue',
+                zone=props.dns_zone
+            ),
+            traffic_encryption=RenderQueueTrafficEncryptionProps(
+                external_tls=RenderQueueExternalTLSProps(
+                    rfdk_certificate=server_cert
+                ),
+                internal_protocol=ApplicationProtocol.HTTPS
+            ),
+            version=self.version,
+            vpc_subnets=render_queue_subnets,
+            vpc_subnets_alb=render_queue_alb_subnets,
+            deletion_protection=False
```

Comment:
We had to add a comment above this in our first example app so that users were aware of the consequences of disabling deletion protection on the RQ, so I wonder if we should add it in here as well.
```
            # TODO - Evaluate deletion protection for your own needs. This is set to false to
            # cleanly remove everything when this stack is destroyed. If you would like to ensure
            # that this resource is not accidentally deleted, you should set this to true.
```
I'm not sure if we need to have it for every example we do though. What do you think?

Reply:
I tried to trim down anything that the Basic example already explains to make it more obvious as to what we're changing here. I do have a comment in the Architecture section of the readme stating that the example assumes some familiarity with the general architecture of an RFDK farm and to refer to the Basic example if they are not.

Reply:
Sounds good. The reference to the Basic example should be good enough.